src/prefixmaps/data/merged.csv

```diff
@@ -6570,6 +6573,7 @@ merged,HPM.PROTEIN,https://identifiers.org/hpm.protein/,prefix_alias,bioregistry
 merged,HPM.PROTEIN,https://identifiers.org/hpm.protein:,prefix_alias,bioregistry
 merged,HPM.PROTEIN,https://n2t.net/hpm.protein:,prefix_alias,bioregistry
 merged,HPM.PROTEIN,https://www.humanproteomemap.org/protein.php?hpm_id=,prefix_alias,bioregistry
+merged,hpo,http://emmo.info/emmo/application/hpo#,prefix_alias,prefixcc

```

Comment:
this one gives me pause - is it going to mess with your ingests, @kevinschaper ?

Reply:
in theory this should not break anything but I'd still like to be careful with prefixcc